docs/deploying/rhoai.md

```diff
@@ -0,0 +1,105 @@
+# Using Red Hat OpenShift AI
+
+[Red Hat OpenShift AI](https://www.redhat.com/en/products/ai/openshift-ai) is a cloud-native AI platform that bundles together many popular model management projects, including [kserve](https://kserve.github.io/website/latest/).
+
+This example shows how to use kserve with RHOAI to deploy a model on OpenShift, using a modelcar image to load the model without requiring any connection to huggingface hub.
+
+## Deploying with kserve
+
+!!! note
+    **Prerequisite**: Ensure that you have a running Kubernetes cluster with RHOAI installed, image pull credentials for `registry.redhat.io/rhelai1`, and Spyre accelerators.
+
+<!-- TODO: Link to public docs for cluster setup -->
+
+1. Create a ServingRuntime to be used for your models
+
+      ```yaml
+        apiVersion: serving.kserve.io/v1alpha1
+        kind: ServingRuntime
+        metadata:
+          name: vllm-spyre-runtime
+          annotations:
+            openshift.io/display-name: vLLM IBM Spyre ServingRuntime for KServe
+            opendatahub.io/recommended-accelerators: '["ibm.com/aiu_pf"]'
+          labels:
+            opendatahub.io/dashboard: "true"
+        spec:
+          multiModel: false
+          supportedModelFormats:
+            - autoSelect: true
+              name: vLLM
+          containers:
+            - name: kserve-container
+              image: quay.io/ibm-aiu/vllm-spyre:latest.amd64
+              args:
+                - /mnt/models
+                - --served-model-name={{.Name}}
+              env:
+                - name: HF_HOME
+                  value: /tmp/hf_home
+                # Static batching configurations can also be set on each InferenceService
+                - name: VLLM_SPYRE_WARMUP_BATCH_SIZES
+                  value: '4'
+                - name: VLLM_SPYRE_WARMUP_PROMPT_LENS
+                  value: '1024'
+                - name: VLLM_SPYRE_WARMUP_NEW_TOKENS
+                  value: '256'
+              ports:
+                - containerPort: 8000
+                  protocol: TCP
+      ```
+
+2. Create an InferenceService for each model you want to deploy. This example demonstrates how to deploy `ibm-granite/granite-3.1-8b-instruct`.
```

Comment:
It might be nice to include a link to Granite here:

```suggestion
2. Create an InferenceService for each model you want to deploy. This example demonstrates how to deploy the [Granite](https://www.ibm.com/granite) model `ibm-granite/granite-3.1-8b-instruct`.
```